scripts/build/gulp/tasks/build.js

```diff
@@ -37,54 +46,40 @@ function createBundle(configName, callback) {
 	});
 }
 
-gulp.task('webpack.prod', callback => {
+gulp.task('build:webpack.prod', callback => {
 	createBundle('webpack.prod', callback);
 });
 
-gulp.task('webpack.dev', callback => {
+gulp.task('build:webpack.dev', callback => {
 	createBundle('webpack.dev', callback);
 });
 
-gulp.task('build:assets', function(callback) {
-	runSequence(
-		'clean-dist',
-		['build-css', 'copy-ckeditor', 'copy-languages', 'copy-svgs'],
-		['copy-demo', 'post-cleanup', ifRelease('minimize-css')],
-		callback
-	);
-});
-
-gulp.task('build', function(callback) {
-	runSequence(
-		'build:assets',
-		[ifRelease('webpack.prod'), 'webpack.dev'],
-		callback
-	);
-});
-
-gulp.task('clean-dist', function() {
+gulp.task('build:clean', function() {
 	// Remove everything inside "dist" recursively, except "dist"
 	// itself, and its .gitignore.
-	return del([
-		path.join(Constants.distFolder, '**'),
-		`!${Constants.distFolder}`,
-		`!${path.join(Constants.distFolder, '.gitignore')}`
-	], {force: true});
+	return del(
+		[
+			path.join(Constants.distFolder, '**'),
+			`!${Constants.distFolder}`,
+			`!${path.join(Constants.distFolder, '.gitignore')}`
+		],
+		{force: true}
+	);
 });
 
-gulp.task('copy-demo', function() {
+gulp.task('build:demo', function() {
 	return gulp
 		.src(path.join(__dirname, '..', 'demo', 'index.html'))
 		.pipe(gulp.dest(Constants.distFolder));
 });
 
-gulp.task('copy-ckeditor', function() {
+gulp.task('build:ckeditor', function() {
 	return gulp
 		.src(path.join(Constants.rootDir, 'lib', 'ckeditor', '/**'))
 		.pipe(gulp.dest(Constants.editorDistFolder));
 });
 
-gulp.task('post-cleanup', function() {
+gulp.task('build:strip', function() {
```

Comment:
Renamed this because I needed to switch everything to "build:*" format, and didn't want something that would clash with "build:clean". Not sure if it's clear, but the meaning here is that "build:clean" cleans out everything from dist but "build:strip" just strips away a subset from it.

Reply:
@wincent that seem reasonable to me